strictequal.js

```diff
@@ -0,0 +1,13 @@
+const strictEqual = (value1, value2) => {
+  let value1type = typeof value1
+  let value2type = typeof value2
+  if (value1type == value2type && value1 == value2) {
+    return true
+  }
+  else {
+    return false
+  }
```

Comment:
Here we are saying if the condition is true return true, if the condition is false return false. This means we can just return the condition since it will be the value we want returned.

```JavaScript
const strictEqual = (value1, value2) => {
    return typeof value1 == typeof value2 && value1 == value2
}
```

Also since then it would be a single line return we could use fat arrow's implicit return

```JavaScript
const strictEqual = (value1, value2) => typeof value1 == typeof value2 && value1 == value2
```